server/chat-plugins/scavengers.ts

```diff
@@ -426,16 +539,16 @@ class ScavengerHunt extends Rooms.RoomGame {
 		this.room.add(message).update();
 	}
 
-	runEvent(event_id, ...args) {
+	runEvent(event_id: string, ...args: any[]) {
 		let events = this.mods['on' + event_id];
 		if (!events) return;
 
-		events = events.sort((a, b) => b.priority - a.priority);
+		events = events.sort((a: any, b: any) => b.priority - a.priority);
```

Comment:
The `: any` should be unnecessary here. Can you revert this and show me the error? Is `events` not being inferred correctly?

```suggestion
		events = events.sort((a, b) => b.priority - a.priority);
```

Reply:
oh lol there is none now - there was at some point when i didnt set everything up at the start